pkgs/tools/graphics/gifski/default.nix

```diff
@@ -7,13 +7,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "gifski";
-  version = "1.13.0";
+  version = "1.31.1";
```

Comment:
btw, I think upstream tagged it incorrectly, it should be `1.13.1` https://github.com/ImageOptim/gifski/blob/1.31.1/Cargo.toml#L13

Reply:
> btw, I think upstream tagged it incorrectly, it should be `1.13.1` [ImageOptim/gifski@`1.31.1`/Cargo.toml#L13](https://github.com/ImageOptim/gifski/blob/1.31.1/Cargo.toml?rgh-link-date=2023-11-29T05%3A05%3A51Z#L13)

Nice catch! #270819